docs/keypairs.md

```diff
@@ -0,0 +1,86 @@
+# Openstack Keypairs
+
+Read more about Openstack keypairs using the [upstream docs](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/keypair.html).
+
+#### List and view Keypairs
+
+``` shell
```

Comment:
Typically we try to prevent copying upstream documentation, is there a doc we can link to rather than duplicating cli commands ?
Same is true for the other following commands

Reply:
Yes, I will re work the page to link to the upstream documentation. My understanding was that we were trying to aggregate the different upstream docs into our own docs. I have no issues reworking this into linking to the upstream docs.

Reply:
Deleted and linked to upstream docs

Reply:
restored the docs